Jenkinsfile

```diff
@@ -96,11 +96,11 @@ pipeline {
       steps{
         script {
           docker.withRegistry( registryUri, registryCredential ) {
-            devImage.push("3.5.0-${env.GIT_HASH}")
-            deplImage.push("3.5.0-${env.GIT_HASH}")
+            devImage.push("4.0-alpha-${env.GIT_HASH}")
+            deplImage.push("4.0-alpha-${env.GIT_HASH}")
```

Comment:
This '4.0-alpha' image tag is being used for the master branch pipeline only.
Why are we merging master branch to v3.5.0?